src/pages/accounts/resetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,84 @@
+import { reset_password } from '../../../constants';
+import { Button, ErrorBox, InputWithLabel } from '../../../components/reusables/Reusable';
+import { useParams } from 'react-router-dom';
+import { useFormik } from 'formik';
+import { useResetPassword } from './hooks';
+
+interface FormValues {
+  password: string;
+  confirmPassword: string;
+}
+
+export default function ResetPassword() {
+  const { token } = useParams<string>();
+  const { handleSubmit } = useResetPassword();
+  const formik = useFormik({
+    initialValues: {
+      password: '',
+      confirmPassword: '',
+    },
+    validate() {
+      const errors: Partial<FormValues> = {};
+
+      if (!formik.values.password) {
+        errors.password = 'Password is required';
+      } else if (
+        !/^(?=.*[a-z])(?=.*[A-Z])(?=.*[0123456789])(?=.*[@$!%*?&])[A-Za-z0123456789@$!%*?&]{8,}$/.test(
+          formik.values.password
+        )
+      ) {
+        errors.password = 'Must be more 8 than characters and containing a number and symbol';
+      }
+
+      if (!formik.values.confirmPassword) {
+        errors.confirmPassword = 'Please confirm your password';
+      } else if (formik.values.password != formik.values.confirmPassword) {
+        errors.confirmPassword = "Passwords don't match";
+      }
+
+      return errors;
+    },
+    onSubmit: (values: { password: string }) => {
+      handleSubmit(values, token || '');
+    },
+  });
+
+  return (
+    <div className="container__box_reset_p">
+      <div className="container__box__text">
+        <h2>{reset_password.title}</h2>
+        <p>{reset_password.text}</p>
+      </div>
+      <form id="form" onSubmit={formik.handleSubmit} noValidate>
```

Comment:
here too, there is `novalidate`, and you can remove these bottom comments if no longer used!